cou/commands.py

```diff
@@ -108,7 +108,6 @@ def get_subcommand_common_opts_parser() -> argparse.ArgumentParser:
     group.add_argument(
         "--quiet",
         "-q",
-        default=False,
```

Comment:
They are indeed redundant when set to `False` with `store_true`. But if leaving `default` here, someone may accidentally change only one of the `default`  and `action` in future and they would get unexpected result. Additonally, if you look at the [official examples](https://docs.python.org/3/library/argparse.html) of `store_true` and `store_false`, they are never used with default values. So I think it would be better to drop the default value definitions.

Reply:
I'm not against it. Solving